core/shared/src/main/scala/fs2/internal/FreeC.scala

```diff
@@ -14,6 +14,11 @@ private[fs2] sealed abstract class FreeC[F[_], +R] {
       case Left(e) => FreeC.Fail(e)
     })
 
+  def flatMap2[R2](f: Either[Throwable, R] => FreeC[F, R2]): FreeC[F, R2] =
```

Comment:
Rename to `transformWith` (matching naming convention used in other types)

Reply:
kk